README.md

```diff
@@ -63,3 +63,21 @@ The charts in the `Incubator` directory allows charts to be shared and improved
 ## Status of the Project
 
 This project is still under active development. If you experience any [issues](https://github.com/nuodb/nuodb-helm-charts/issues) please let us know, or better yet, contribute a fix or feature.
+
+## Testing
+
+To run the tests in this project, Go 1.12 is required.
+The easiest way to clone the repository and run a test suite (e.g. `test/integration`) is as follows:
+
+```
+GO111MODULE=off go get github.com/nuodb/nuodb-helm-charts
+cd $(go env GOPATH)/src/github.com/nuodb/nuodb-helm-charts
+go test -v -timeout 60s ./test/integration
+```
+
+The tests in `test/minikube` require Minikube to be running on the local host and can be run as follows:
```

Comment:
It actually can be any Kubernetes cluster. The only requirement is that the `kubectl` is configured with the required testing cluster.